### PR TITLE
Current requests

### PR DESCRIPTION
**Pull request by deepali1-aconnexion**

### CODE REVIEW

File: app/views/current_requests/_show.html.slim [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+            |By Accepting you agree to out 
+            = link_to 'Terms of use', '#'
+          .hidden
+            .bg-red.w100-pct.p10.white.mt20
+                | Authorize Payment
+            .text-center
+              strong Pick a payment method:
+              = link_to 'Decline', '#', class: 'btn btn-default w100-pct text-center mt10'
+              br/
+              = link_to 'Pay $10.00', '#', class: 'btn btn-danger w100-pct text-center mt20'
```
Comment (AlexanderUlitin) [inline app/views/current_requests/_show.html.slim]: @deepali1-aconnexion this hardcoded amount of Diagnostic fee will be changed in future pull-requests?

File: app/views/current_requests/_show.html.slim [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+          .black.text-center.fs16
+            |You must authorize a charge of $10,00
+            .mt10
+              | This covers the technician's arrival and one hour of br/ work to fix or diagnose the problem. Once you br/ accept this conctractor you cannot cancel and your br/ card will be charged when the technician arrives.
+          = link_to 'Accept and authorize charge', '#', remote: true, class: 'btn btn-danger w100-pct text-center mt20'
+          br/
+          = link_to 'Decline', account_service_request_assignment_decline_reasons_path(account, service_request, assignment), remote: true, class: 'btn btn-default w100-pct text-center mt10 with-ajax-loader'
+          p.text-center.fs12.mt10
+            |By Accepting you agree to out 
+            = link_to 'Terms of use', '#'
```
Comment (AlexanderUlitin) [inline app/views/current_requests/_show.html.slim]: @deepali1-aconnexion it should probably links to some page with terms?

File: app/views/current_requests/_show.html.slim (hunk 1 above)
Comment (deepali1-aconnexion) [reply app/views/current_requests/_show.html.slim]: Added diagnostic fee.

File: app/views/current_requests/_show.html.slim (hunk 2 above)
Comment (deepali1-aconnexion) [reply app/views/current_requests/_show.html.slim]: Added Terms of use page.

**Comment by deepali1-aconnexion** [comment]
Added humanized_money_with_symbol helper in views.

**Pull request merged**